backbone.js

```diff
@@ -644,7 +646,7 @@
       if (attrs && wait) this.attributes = _.extend({}, attributes, attrs);
 
       var method = this.isNew() ? 'create' : (options.patch ? 'patch' : 'update');
-      if (method === 'patch' && !options.attrs) options.attrs = attrs;
+      if (method === 'patch' && !options.attrs) options.attrs = this.changedAttributes(attrs);
```

Comment:
That's not how this method is supposed to work but the rest of the pull is great. Back out this change, and I'll merge. Thanks!